rdbi-parent/pom.xml

```diff
@@ -215,8 +215,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>11</source>
+                    <target>11</target>
```

Comment:
could probably still build this for 1.8, otherwise we'll need to maintain a separate java11+ release branch